apps/web/src/services/administration/iam/user/modules/UserManagementTable.vue

```diff
@@ -347,7 +347,7 @@ export default {
                     await bindRole(item.user_id, roleId);
                     userFormState.roleOfSelectedUser = roleId;
                 }
-                if (roleId === '' && userFormState.roleOfSelectedUser !== '') {
+                if (roleId === '' || roleId === null && userFormState.roleOfSelectedUser !== '') {
```

Comment:
question: Is there a reason why you didn't use `!roleId`?😮  If so, please ignore this comment.